### PR TITLE
Craftable Mags

### LINKED ISSUE #1464: craftable magazines
for like

- [ ] 7.62mm bullets
- [ ] 5.56mm bullets
- [ ] 9x18mm bullets 

to start with

just require some 'sheet metal' to craft

### PR DESCRIPTION
**Pull request by maxgestic**
Adds the ability to craft magazines for AP's, Carbine's and AK's

Closes #1464

### CODE REVIEW

**Comment by maxgestic** [comment]
closes https://github.com/minipunch/usarealismrp-fxserver/issues/1464

**Comment by maxgestic** [comment]
closes #1464

File: resources/usa-crafting/config.lua [hunk 1]
```diff
@@ -753,6 +753,75 @@ Config.recipes = {
             type = "ammo",
             craftDurationSeconds = 3 * 60
         },
+        { 
+            name = "30 round Magazine (7.62mm)",
```
Comment (minipunch) [inline resources/usa-crafting/config.lua:757]: U sure the item image works with this name?

Comment (maxgestic) [reply resources/usa-crafting/config.lua:757]: Yea cos its just the image in the crafting dialog

Comment (minipunch) [reply resources/usa-crafting/config.lua:757]: Oh ye true

**Review by minipunch — APPROVED** [review]

**Pull request merged**